lib/sem/cli.rb

```diff
@@ -8,10 +8,19 @@ class CLI < Dracula
     require_relative "cli/shared_configs"
 
     desc "login", "log in to semaphore from the command line"
-    def login(auth_token)
+    option :auth_token, :required => true
+    def login
+      auth_token = options[:auth_token]
+
       Sem::Credentials.write(auth_token)
 
-      puts "Your credentials have been saved to #{Sem::Credentials::PATH}"
+      orgs = Sem::API::Orgs.list
```

Comment:
@ijovan can you help me with this? How can I test this out in a more streamlined fashion?